jdaviz/configs/default/plugins/about/about.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <j-tray-plugin
+    :description="docs_description || 'About Jdaviz.'"
+    :popout_button="popout_button"
+    :scroll_to.sync="scroll_to">
+
+    <v-row>
+      <v-text-field class="v-messages v-messages__message text--secondary"
+        v-model="jdaviz_version"
+        label="Version"
+        hint="Version of installed Jdaviz."
+        readonly>
+      </v-text-field>
+    </v-row>
+
+    <v-row v-if="not_is_latest">
+      <span class="v-messages v-messages__message text--secondary" style="color: red !important">
+        A newer version ({{ jdaviz_pypi }}) is available from PyPI.
+      </span>
+      <j-docs-link link="https://pypi.org/project/jdaviz/" linktext="Go to PyPI">
+        Please update Jdaviz and restart your session.
+      </j-docs-link>
+
+    </v-row>
+
+    <v-row>
+      <j-docs-link link="https://github.com/spacetelescope/jdaviz/blob/main/CHANGES.rst" linktext="Change Log">
+        To see list of changes, go to your version section in the change log.

```

Comment:
this ends up resulting in change log twice, what about the following?
```suggestion
      <j-docs-link link="https://github.com/spacetelescope/jdaviz/blob/main/CHANGES.rst" linktext="Change Log">
        To see list of changes, go to your version section in the

```

Can/should we link directly to the version?  Or maybe we don't even need "go to your version section" at all (if the intention is to see changes _since_ the installed version or if we think it's just self-explanatory)?

Reply:
Linking directly to version will need some regex magic. It is possible but not something I want to break my brain over for this ticket. Maybe we can start from the logic here if you really want it: https://github.com/scientific-python/action-check-changelogfile/blob/main/rst_parser.py

Reply:
definitely not necessary now if its not trivial (and since there is no API, super easy to iterate on things later)